docs/source/error_code_list.rst

```diff
@@ -908,6 +908,24 @@ Functions will always evaluate to true in boolean contexts.
     if f:  # Error: Function "Callable[[], Any]" could always be true in boolean context  [truthy-function]
         pass
 
+Check that function isn't used in boolean context [str-bytes-safe]
+-------------------------------------------------------------------
+
+Warn about dangerous coercions related to bytes and string types.
```

Comment:
Not sure "dangerous" is the right word since it won't crash, just give potentially unexpected output.

Reply:
I feel like the theme here is "implicit" which makes the intention ambiguous.

Reply:
Agree, "implicit" is a good word to use